content/en/project/eeg_preprocessing/index.md

```diff
@@ -0,0 +1,43 @@
+# BrainHack School 2023 Project 
```

Comment:
Please incorporate the following header at the beginning of your project, and of course update the information: 

``` 
---
type: "project" # DON'T TOUCH THIS ! :)
date: "2020-05-16" # Date you first upload your project.
# Title of your project (we like creative title)
title: "This is an example project page which serves as a template"

# List the names of the collaborators within the [ ]. If alone, simple put your name within []
names: [Samuel Guay, Pierre Bellec]

# Your project GitHub repository URL
github_repo: https://github.com/brainhack-school2023/project_template

# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
website:

# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/brainhack-school2020/project_template), click `manage topics`.
# Please only lowercase letters
tags: [project, github, markdown, brainhack]
```